modules/services/protonmail-bridge.nix

```
@@ -0,0 +1,71 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.protonmail-bridge;
+in
+{
+  meta.maintainers = with lib.hm.maintainers; [ epixtm ];
+
+  options.services.protonmail-bridge = {
+    enable = lib.mkEnableOption "ProtonMail Bridge";
+    package = lib.mkPackageOption pkgs "protonmail-bridge" { };
+
+    path = lib.mkOption {
```

Was about to merge, but think this option name is a little confusing. 

```suggestion
    extraPackages = lib.mkOption {
```

---

Should packages added to `extraPackages` also be added to the user's home profile? I'm not certain of the convention.